presto-product-tests/bin/product-tests-suite-8-non-generic.sh

```diff
@@ -12,7 +12,7 @@ fi
 
 suite_exit_code=0
 
-TESTS_HIVE_VERSION_MAJOR="3" TESTS_HIVE_VERSION_MINOR="1" presto-product-tests-launcher/bin/run-launcher test run \
```

Comment:
This belongs to `Use product version obtained from Hive in Hive product tests` commit.